js/ui/codeView.js

```diff
@@ -405,6 +405,9 @@ var CodingSession = new Lang.Class({
         this._hackablePropsChangedId = 0;
         this._hackable = true;
 
+        this._appParent = null;
+        this._appPositionId = 0;
```

Comment:
**Nitpick:** When I saw this, I was wondering what `position ID` was; it'd be a bit more clear if it were called `this._appPositionHandler` or `this._appPositionHandlerId`. But it's a mere suggestion (really up to you).